examples/inverse_api.rs

```diff
@@ -78,7 +78,7 @@ fn lines<'l, Tx: IntoIterator<Item = f32>, Ty: IntoIterator<Item = f32>>(x: Tx,
 
 impl Lines
 {
-	fn show(&self)
```

Comment:
Why'd you need to change this?

Reply:
I must have changed this to suppress a cargo warning. I've reverted it.